core/themes/seven/style.css

```diff
@@ -181,13 +181,20 @@ ul.menu li.expanded {
 /**
  * Tabs.
  */
+@media only screen and (max-width: 600px) {
+  .tabs {
+    clear: both;
+  }
+}
+
```

Comment:
@quicksketch 
Issue on how we should handle media query fallbacks:
https://github.com/backdrop/backdrop-issues/issues/801